external/deep-object-reid/torchreid_tasks/openvino_task.py

```diff
@@ -193,7 +193,7 @@ def infer(self, dataset: DatasetEntity,
             probs_meta = TensorEntity(name="probabilities", numpy=probs.reshape(-1))
             dataset_item.append_metadata_item(probs_meta, model=self.model)
 
-            feature_vec_media = TensorEntity(name="representation_vector", numpy=repr_vector.reshape(-1))
+            feature_vec_media = TensorEntity(name="representation_vector", numpy=np.copy(repr_vector.reshape(-1)))
```

Comment:
Why is the `copy()` necessary for `repr_vector` but not for `probs` in the previous line? Can you maybe add a comment to the code, it isn't obvious that "memory is shared for all feature vectors in numpy". Also, can the copy be moved into the `inferencer.predict` function, to prevent developers from forgetting to copy the output?

Reply:
@leoll2 yeah, I agree, you'd need to run debugger identifying this issue. I will move the vector copying to `TensorEntity`, and add a comment there